### PR TITLE
Fix typename diagnostics

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR:
  * Improves diagnostic messages in the `TypeUrl` class.
  * Temporarily disables some plug-in tests that result in out-of-memory build errors.

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@armiol, PTAL.

File: .idea/.name [hunk 1]
```diff
@@ -1 +1 @@
-spine-base
\ No newline at end of file
+spine
```
Comment (armiol) [inline .idea/.name]: I think the change in this file should be reverted, as we have renamed the root project to `spine-base` intentionally.

File: base/src/main/java/io/spine/protobuf/MessageFieldException.java [hunk 2]
```diff
@@ -12,28 +13,31 @@
...
+     *        a format string for the error message
+     * @param params
+     *        error message parameters
      */
     public MessageFieldException(Message protobufMessage,
                                  String errorMessageFormat,
                                  Object... params) {
         super(format(checkNotNull(errorMessageFormat), params));
-        this.protobufMessage = checkNotNull(protobufMessage);
+        this.protobufMessage = (GeneratedMessageV3)checkNotNull(protobufMessage);
```
Comment (armiol) [inline base/src/main/java/io/spine/protobuf/MessageFieldException.java]: A single-space character seems to be missing after `(GeneratedMessageV3)`.

File: base/src/main/java/io/spine/type/KnownTypes.java [hunk 3]
```diff
@@ -253,13 +253,17 @@ public boolean apply(@Nullable TypeUrl input) {
      * @throws IllegalArgumentException if the name does not correspond to any known type
      */
     static GenericDescriptor getDescriptor(String typeName) {
+        checkNotNull(typeName);
+        checkArgument(!typeName.isEmpty(), "Type name cannot be empty");
```
Comment (armiol) [inline base/src/main/java/io/spine/type/KnownTypes.java]: Seems that `Validate.checkNotEmptyOrBlank` may fit better than both of `check...` statements.

**Pull request merged**